<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChildrenType</class>
 <widget class="QMainWindow" name="ChildrenType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>584</width>
    <height>505</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>584</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChildrenType</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Jadid</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Father</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbHair_F">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Straight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Curly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wavy</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hair type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbEye_F">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eye color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbSkin_F">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Skin color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbEye_M">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbSkin_M">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Skin color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbHair_M">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Straight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Curly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wavy</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eye color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hair type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Jadid</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mother</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCheck">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>281</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro Semibold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtOutput_number">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Numbers</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBlood_F">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>O+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB-</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blood type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBlood_M">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>O+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB-</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blood type</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
